pipelinewise/cli/utils.py

```diff
@@ -181,42 +186,37 @@ def load_yaml(yaml_file, vault_secret=None):
         secret_file.load()
         vault.secrets = [('default', secret_file)]
 
-    # YAML ENV VAR
```

Comment:
There was a conflict in this file. It looks like the way the YAML configuration is loaded was changed. The changes we made were to enable reading environment variables when loading the config, but it seems pipelinewise has added equivalent functionality internally (https://transferwise.github.io/pipelinewise/installation_guide/creating_pipelines.html#environment-variables-in-yaml-config). I'm submitting a PR to change the syntax in datawarehouse.

Reply:
Just to be clear, this file is now identical to upstream, I removed any changes we had made.